.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: haskell
+
+ghc:
+  - "8.2"
+  - "8.0"
+  - "7.10"
```

Comment:
Not sure if we want to test backwards compatibility here... I think we better stick to one version...